paper/paper.md

```diff
@@ -0,0 +1,70 @@
+
+While some research code for these models exists, they are not focused on usability and are limited to FORTRAN **TODO LINK** and proprietary software like MATLAB **TODO LINK**. There is a lack of well-documented, actively maintained open-source implementations, particularly in modern programming languages, and with a focus on scalability and performance.
+
+The presented software has already seen application in several research works. It has been used to optimize trajectories around the highly irregular comet 67P/Churyumov-Gerasimenko [@marak2023trajectory]. Further, it has been used to study the effectiveness of so-called neural density fields [@izzo2022geodesy], where it can serve as ground truth and to pre-train neural networks [@GeodesyNetsBenchmark]. **TODO_add_more_examples**
+
+Thus, overall this model is highly versatile and can be used in a wide range of applications. We hope it will enable further research in the field, especially related to recent machine learning techniques, which typically rely on Python implementations.
+
+# Polyhedral Model
+
+On a mathematical level, the implemented model follows the approach by Petrović [@petrovic1996determination] as refined by Tsoulis and Petrović [@tsoulis2001singularities]. A comprehensive description of the mathematical foundations of the model is given in the associated student report [@jonas_report]. 
+
+Implementation-wise, it makes use of the inherent parallelization opportunity of the approach as it iterates over the mesh. This parallelization is achieved via *thrust*, which allows utilizing *OpenMP* and *Intel TBB*. On a finer scale, individual, costly operations were investigated and, e.g., the \texttt{arctan} operations were vectorized using *xsimd*. On an application side, the user may use the implemented caching mechanism to avoid the recomputation of mesh properties, such as the face normals.
+
+Extensive tests using GoogleTest are used via GitHub Actions to ensure the (continued) correctness of the implementation.
+
+# Installation \& Contribution
+
+The library is available on GitHub \footnote{\url{https://github.com/esa/polyhedral-gravity-model}} and can be installed with *pip* or from *conda* \footnote{\url{https://anaconda.org/conda-forge/polyhedral-gravity-model}}. Build instructions using *CMake* are provided in the repository. The library is licensed under a GPL license.
```

Comment:
Maybe stupid question but is it available / tested for all relevant operating systems ?

Reply:
Available: Yes: Windows 64, Linux x64 and ARM64 (wheel only for x64 for the recent version 2.1), OSX x64 and ARM64 (wheels only for the x64 for the recent version 2.1) - for each CPython since 3.6 and PyPy since 3.7

Tested: Import Checks are conducted for everything—proper CI Testing only on Linux due to performance reasons. However, I ran the test irregularly in my setup on Windows and OSX.